### DISCUSSION TITLE
about my Tiger Lake iGPU

### DISCUSSION THREAD

**Discussion by Bronstein0x113c1c3** [lshbluesky/Samsung-NT750XDA-KF59U-Hackintosh#29] [Q&A] [answered]
I tend to install MacOS on my computer andI have seen your Hackintosh config and I think it's so impossible to do that. I use Intel 11400H, with "nearly" Iris Xe 9a68, can you give me some information about the AAPL,snb-platform-id, I have tried too many times but it's still laggy....

Many thanks from a Tiger Lake user.
Have a good day.
![image](https://github.com/user-attachments/assets/6fd427ae-9e9e-4f09-8550-cec5f52aab7f)

**Comment by lshbluesky** [comment, marked as answer]
Hello.
Sorry for replying after almost 7 months.
The Tiger Lake iGPU with Device ID `0x9A68` has already been fixed to load from my modified WhateverGreen.kext.

And, the correct Platform ID property name is `AAPL,ig-platform-id`, not `AAPL,snb-platform-id`.
`AAPL,snb-platform-id` is a property name used only by Intel 2nd Gen Sandy Bridge iGPUs.

However, regardless of Device ID, it is still not possible to enable QE/CI graphics acceleration on Tiger Lake iGPU.
This research has been discontinued because it is very difficult to get Accelerator Kext drivers (AppleIntelICLGraphics.kext, AppleIntelTGLGraphics.kext) to work properly on Tiger Lake iGPU to enable Metal acceleration.
And, I'm sorry, but I don't currently have a Tiger Lake laptop. 😔

Have a good day. 🍀
Sincerely.

**Reply by hla63** [reply]
> Hello. Sorry for replying after almost 7 months. The Tiger Lake iGPU with Device ID `0x9A68` has already been fixed to load from my modified WhateverGreen.kext.
> 
> And, the correct Platform ID property name is `AAPL,ig-platform-id`, not `AAPL,snb-platform-id`. `AAPL,snb-platform-id` is a property name used only by Intel 2nd Gen Sandy Bridge iGPUs.
> 
> However, regardless of Device ID, it is still not possible to enable QE/CI graphics acceleration on Tiger Lake iGPU. This research has been discontinued because it is very difficult to get Accelerator Kext drivers (AppleIntelICLGraphics.kext, AppleIntelTGLGraphics.kext) to work properly on Tiger Lake iGPU to enable Metal acceleration. And, I'm sorry, but I don't currently have a Tiger Lake laptop. 😔
> 
> Have a good day. 🍀 Sincerely.

https://github.com/pawan295/Appleinteltgldriver.kext 
work in progress
